cmd/dagger/module_inspect.go

```diff
@@ -97,28 +97,31 @@ func tryInitializeModule(ctx context.Context, dag *dagger.Client, srcRef string)
 	defer telemetry.End(span, func() error { return rerr })
 
 	findCtx, findSpan := Tracer().Start(ctx, "finding module configuration", telemetry.Encapsulate())
-	conf, _ := getModuleConfigurationForSourceRef(findCtx, dag, srcRef, true, true)
+	modSrc := dag.ModuleSource(srcRef, dagger.ModuleSourceOpts{
+		DisableFindUp: true,
+	})
```

Comment:
The previous call to `getModuleConfigurationForSourceRef` had "find up" turned on, but this disables it.

Reply:
I added this while fixing some failing tests, but I dumbly did not add a comment here on *why* it seemed necessary... I will try rm'ing it and see what, if any, tests fail and figure it out from there. I agree it probably makes more logical sense to enable it here for consistency

Reply:
All the tests passed w/ it enabled so yeah I guess this wasn't necessary in the end, re-enabled it